lprint-brother.c

```diff
@@ -449,6 +447,13 @@ lprint_brother_rstartjob(
   else if (darkness > 100)
     darkness = 100;
 
+  // 64 == auto-cut
+  // TODO: Is this not reversed? Continuous should mean *no* autocut?
```

Comment:
"continuous" is the media type, which means that instead of discrete/separated labels there is a continuous ribbon/tape. We don't need to cut and already cut label, but we do need to cut the continuous ribbon/tape.

Reply:
Ah, that makes sense. I was somehow thinking this was a setting for "cut labels", or "leave them uncut" (aka chain print), but now I see this is something different (and that such a setting is not yet implemented, so that's a separate topic).